wire-ios-data-model/Tests/Source/Model/ConversationList/ZMConversationListDirectoryTests.m

```diff
@@ -65,9 +65,20 @@ - (void)setUp
     Team *team = [Team insertNewObjectInManagedObjectContext:self.uiMOC];
     team.remoteIdentifier = [NSUUID createUUID];
 
+    ZMUser *selfUser = [ZMUser selfUserInContext:self.uiMOC];
+    selfUser.remoteIdentifier = [NSUUID createUUID];
```

Comment:
Do we need to use `createUUID` or could it be `[NSUUID new]` aka `[[NSUUID alloc] init]`?